### PR TITLE
Fixing parsing of device Ids

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: model/push_notification_test.go [hunk 1]
```diff
@@ -17,3 +17,78 @@ func TestPushNotification(t *testing.T) {
 		t.Fatal("Ids do not match")
 	}
 }
+
+func TestPushNotificationDeviceId(t *testing.T) {
+
+	msg := PushNotification{Platform: "test"}
+
+	msg.SetDeviceIdAndPlatform("android:12345")
```
Comment (enahum) [inline model/push_notification_test.go]: can we add test cases for android2, apple and apple2

Comment (enahum) [inline model/push_notification_test.go]: I guess is android_rn and apple_rn

**Comment by grundleborg** [comment]
This PR appears to be related: https://github.com/mattermost/platform/pull/5573

File: model/push_notification_test.go (hunk 1 above)
Comment (coreyhulen) [inline model/push_notification_test.go]: The platform doesn't matter anymore.  It can be set to anything.  It only matters in the push proxy.

**Comment by coreyhulen** [comment]
yeah, look like #5573 fixes the same issue.

**Pull request merged**